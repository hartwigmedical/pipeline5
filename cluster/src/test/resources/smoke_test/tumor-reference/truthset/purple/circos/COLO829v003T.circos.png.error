Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781, <F> line 31.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781, <F> line 33.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781, <F> line 335.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
Use of uninitialized value in subroutine entry at /opt/tools/circos/0.69.6/bin/../lib/Circos/Configuration.pm line 781.
