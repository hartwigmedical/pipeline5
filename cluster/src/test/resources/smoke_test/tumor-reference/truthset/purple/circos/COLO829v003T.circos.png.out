debuggroup summary 0.24s welcome to circos v0.69-6 31 July 2017 on Perl 5.024001
debuggroup summary 0.24s current working directory /
debuggroup summary 0.24s command /opt/tools/circos/0.69.6/bin/circos -nosvg -conf /data/output/circos/COLO829v003T.circos.conf -outputdir /data/output/plot -outputfile COLO829v003T.circos.png
debuggroup summary 0.24s loading configuration from file /data/output/circos/COLO829v003T.circos.conf
debuggroup summary 0.24s found conf file /data/output/circos/COLO829v003T.circos.conf
debuggroup summary 0.46s debug will appear for these features: output,summary
debuggroup summary 0.46s bitmap output image /data/output/plot/COLO829v003T.circos.png
debuggroup summary 0.46s parsing karyotype and organizing ideograms
debuggroup summary 0.60s karyotype has 24 chromosomes of total size 3,095,677,436
debuggroup summary 0.61s applying global and local scaling
debuggroup summary 0.62s allocating image, colors and brushes
debuggroup summary 9.46s drawing 24 ideograms of total size 3,095,677,436
debuggroup summary 9.46s drawing highlights and ideograms
debuggroup summary 9.46s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/highlight.bg.conf
debuggroup summary 9.46s process track_0 highlight /data/output/circos/gaps.txt
debuggroup summary 9.56s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/link.conf
debuggroup summary 9.56s process track_0 link /data/output/circos/COLO829v003T.link.circos
debuggroup summary 9.56s drawing link track_0 z 0
debuggroup summary 9.57s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/scatter.conf
debuggroup summary 9.57s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/heatmap.conf
debuggroup summary 9.57s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/scatter.conf
debuggroup summary 9.57s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/histogram.conf
debuggroup summary 9.57s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/histogram.conf
debuggroup summary 9.57s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/scatter.conf
debuggroup summary 9.57s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/histogram.conf
debuggroup summary 9.57s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/histogram.conf
debuggroup summary 9.57s processing track_0 scatter /data/output/circos/COLO829v003T.snp.circos
debuggroup summary 9.57s processing track_1 heatmap /data/output/circos/COLO829v003T.indel.circos
debuggroup summary 9.58s processing track_2 scatter /data/output/circos/COLO829v003T.cnv.circos
debuggroup summary 9.61s processing track_3 histogram /data/output/circos/COLO829v003T.cnv.circos
debuggroup summary 9.63s processing track_4 histogram /data/output/circos/COLO829v003T.cnv.circos
debuggroup summary 9.65s processing track_5 scatter /data/output/circos/COLO829v003T.map.circos
debuggroup summary 9.67s processing track_6 histogram /data/output/circos/COLO829v003T.map.circos
debuggroup summary 9.69s processing track_7 histogram /data/output/circos/COLO829v003T.map.circos
debuggroup summary 9.70s drawing track_0 scatter z 0 COLO829v003T.snp.circos orient out
debuggroup summary 9.70s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 9.85s drawing track_1 heatmap z 0 COLO829v003T.indel.circos 
debuggroup summary 9.85s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 9.90s drawing track_2 scatter z 0 COLO829v003T.cnv.circos orient out
debuggroup summary 9.90s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 9.90s drawing track_3 histogram z 0 COLO829v003T.cnv.circos orient out
debuggroup summary 9.90s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 10.08s drawing track_4 histogram z 0 COLO829v003T.cnv.circos orient out
debuggroup summary 10.08s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 10.21s drawing track_5 scatter z 0 COLO829v003T.map.circos orient out
debuggroup summary 10.21s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 10.21s drawing track_6 histogram z 0 COLO829v003T.map.circos orient out
debuggroup summary 10.21s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup summary 10.32s drawing track_7 histogram z 0 COLO829v003T.map.circos orient out
debuggroup summary 10.32s found conf file /opt/tools/circos/0.69.6/bin/../etc/tracks/axis.conf
debuggroup output 10.41s generating output
debuggroup output 10.90s created PNG image /data/output/plot/COLO829v003T.circos.png (564 kb)
